src/server/app.ts

```diff
@@ -1,17 +1,17 @@
-import fs from 'fs';
+import * as fs from 'fs';
```

Comment:
I don't think these changes are necessary.

I have the branch checked out, and this is fine: 
`git checkout master src/server/app.ts && docker-compose run --rm node npm run build-server`

Reply:
done

Reply:
Sorry, I meant that none of the changes in `src/server/app.ts` are needed.

Reply:
done